rootdir/vendor/etc/camera/camera_config.xml

```diff
@@ -23,6 +23,30 @@
       <MinFocusDistance>0.1</MinFocusDistance>
     </LensInfo>
   </CameraModuleConfig>
+  <CameraModuleConfig>
+    <CameraId>1</CameraId>
+    <SensorName>s5k4h8</SensorName>
+    <ActuatorName>lc898214xc</ActuatorName>
+    <FlashName>pmic</FlashName>
+    <ChromatixName>s5k4h8_chromatix</ChromatixName>
+    <ModesSupported>1</ModesSupported>
+    <Position>BACK_AUX</Position>
```

Comment:
BACK!! NOT BACK_AUX!!

Reply:
Ok. Still should not change that much. And surely shouldn't stop probing of it.

Reply:
Will change tomorrow when I get home

Reply:
Won't stop it from probing but you won't be able to use the camera normally, so, only as an auxiliary cam (for pdaf, for tof, as a b/w camera etc, which is not what you want).